frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx

```diff
@@ -205,7 +205,6 @@ const UploadAdministrationData = () => {
         .then(() => {
           setLoading(false);
           formRef.current.resetFields();
-          navigate("/control-center/master-data/download-administration-data");
```

Comment:
I guess this will depend on this PR
https://github.com/akvo/rtmis/pull/1236

To handle redirects after successfully downloading a prefilled administrative list